.terraform/test.sh

```diff
@@ -0,0 +1,24 @@
+#
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+scriptDir="$(cd -P -- "$(dirname -- "${BASH_SOURCE[0]}")" && pwd -P)"
+pushd "$scriptDir" >/dev/null || exit
+
+source ./init.sh "$1"
```

Comment:
How might we run all (applicable) modules that has terraform configuration?

Reply:
As described in `./.terraform/README.md`, running `./.terraform/test.sh` without any arguments will perform integration tests on all modules. If a module has a terraform configuration associated with it in `./<module>/.terraform`, then its configuration will be incorporated in the requirements while setting up the project.

There isn't a command to perform integration testing only on modules with terraform configurations, though I could make that happen if that's something you'd interested in.

Alternatively, `./.terraform/test.sh <module>:<module>` will test the given modules specifically.